core/templates/pages/exploration-editor-page/preview-tab/preview-tab.component.html

```diff
@@ -29,6 +29,18 @@
 </div>
 
 <style>
+  /*
+  Note that this affects both the learner mode and the 'editor preview'
+  mode.
+  */
+  html, body {
+    background: #e8e7e3 no-repeat center center fixed;
+    background-size: cover;
+    color: rgba(0,0,0,0.87);
+    font-family: 'Roboto', Arial, sans-serif;
+    font-size: 1.0em;
+  }
+
```

Comment:
Why did we not need this before the migration?

Reply:
Earlier these styles were part of `conversation-skin.directive.ts`. So, these styles gets applied to wherever it is loaded (exploration player page and preview tab of exploration editor page). Unlike AngularJs directives, angular component have view encapsulation(that is the component styles are not applied outside the component's template). 
So, to get the same as before UI, I had two choices.
1. Disable view encapsulation of conversation-skin.component.ts
2. Move these styles to preview-tab.directive.ts where there is no view encapsulation because it is in angularjs.

We try to minimize the use of ViewEncapsulation.None
So, I went with the second approach.

Reply:
Perhaps you could leave a comment explaining that this doesn't affect other files due to view encapsulation in Angular. We can remove such comments once all files are migrated.